conf.py

```diff
@@ -69,21 +38,14 @@
 html_static_path = ['_static']
 templates_path = ['_templates']
 
-html_sourcelink_suffix = ''
 exclude_patterns = ['*.ipynb', '_build', 'legacy_tutorials',
-                    '**.ipynb_checkpoints']
+                    '**.ipynb_checkpoints', '.tox']
 
-cell_timeout = int(os.getenv('QISKIT_CELL_TIMEOUT', 180))
-nbsphinx_timeout = cell_timeout
+nbsphinx_timeout = 300
 nbsphinx_execute = 'always'
-nbsphinx_execute_arguments = [
-    "--InlineBackend.figure_formats={'png', 'pdf'}",
-    "--InlineBackend.rc={'figure.dpi': 96}",
-]
-
-nbsphinx_thumbnails = {
-}
-
+nbsphinx_widgets_path = ""
+html_sourcelink_suffix = ""
+nbsphinx_thumbnails = {"**": "_static/no_image.png"}
```

Comment:
I aligned this config with qiskit-metapackage, since that is what actually ends up being used in the final rendered docs.

Reply:
Great! This solve the thumbnail issue in the local build. In general, it's nice to see the same output in local build of the tutorials repo alone as the final rendered docs built by the metapackage.